### ISSUE TITLE
Enhancement - Remove camera overlays

### ISSUE THREAD

**Issue by jfrazierjr** [Oromis/touch-vtt#48]
I am not sure if this locked to the main application or if a hook can be added in to remove the camera overlays.    Our group is planning to stream some players and want video, however ONE person is on mobile only and the camera overlay just completely cuts out a huge portion of the already small phone screen.

Hopefully, you can see just how much of the screen this covers up in the below screenshot.  Even this minimal is only AFTER clicking the slider button to disconnect from the Livekit server(thats what I am running but I assume the same would be with Foundry's built in peer to peer video connection since livekit is just the data transfer parts).    Prior to disconnecting from livekit, I had up to three camera overlays showing up!  

![Screenshot_20220825-234014_Chrome](https://user-images.githubusercontent.com/2130984/186812709-5270bdd9-6a78-41f7-81be-d884b849b5be.jpg)

Id be happy to assist in any testing if this is something the author could possible fix via a global setting(default to off perhaps??????)

**Comment by Aioros** [comment]
I'm closing this as I feel it's out of scope for TouchVTT. It might be more appropriate for something like Mobile Improvements.
